docs/engineering/technical-planning.md

```diff
@@ -0,0 +1,36 @@
+# Technical Planning Docs
+## Request for Comment
+In the course of planning development, there are often scenarios where a technology or implementation decision fork is presented and an engineer wants to solicit ideas or feedback. 
```

Comment:
perhaps reword the end of this sentence to be "... and an engineer wants to solicit feedback for an idea or problem-set."

Reply:
Eh, I don't think that changes the meaning of the sentence.